yt/data_objects/tests/test_particle_filter.py

```diff
@@ -33,6 +33,51 @@ def stars(pfilter, data):
     ad['deposit', 'stars_cic']
     assert True
 
+
+def test_add_particle_filter_overriding():
+    """Test the add_particle_filter overriding"""
+    from yt.data_objects.particle_filters import filter_registry
+    from yt.funcs import mylog
+
+    def star_0(pfilter, data):
+        pass
+
+    def star_1(pfilter, data):
+        pass
+
+    # Use a closure to store whether the warning was called
+    def closure(status):
+        def warning_patch(*args, **kwargs):
+            print('I am called!')
```

Comment:
this print statement needs to be removed

Reply:
My bad, I removed it.